bundles/org.openhab.binding.velbus/README.md

```diff
@@ -1,140 +1,267 @@
 
-The binding has been tested with a USB configuration module for universal mounting (VMB1USB).
-For optimal stability, the preferred configuration module is the VMBRSUSB module.
+For optimal stability, the preferred configuration module is the VMBRSUSB module. Consider
+deploying a TCP bridge – the officially developed [python-velbustcp] or one of the [third party
+projects][3rd-party-servers] – in between the configuration module and OpenHAB. Doing so allows
+concurrent access to your Velbus system by both OpenHAB and the official configuration software.
 
-The binding exposes basic actions from the Velbus System that can be triggered from the smartphone/tablet interface, as defined by the [Velbus Protocol info sheets](https://github.com/velbus).
+[python-velbustcp]: https://github.com/velbus/python-velbustcp
+[3rd-party-servers]: https://github.com/StefCoene/velserver/wiki/TCP-server-for-Velbus
 
-Supported item types are switches, dimmers and rollershutters.
-Pushbutton, temperature sensors and input module states are retrieved and made available in the binding.
+The binding exposes basic actions from the Velbus System that can be triggered from the
+smartphone/tablet interface, as defined by the [Velbus Protocol info
+sheets](https://github.com/velbus).
+
+Supported item types are switches, dimmers and rollershutters. Pushbutton, temperature sensors and
+input module states are retrieved and made available in the binding.
 
 ## Supported Things
 
-A Velbus configuration module (e.g. VMBRSUSB) or a network server (e.g. [VelServer](https://github.com/StefCoene/velserver/wiki/TCP-server-for-Velbus)) is required as a "bridge" for accessing any other Velbus devices.
+In addition to the bridge modules mentioned in the section above, the supported Velbus modules are:
 
-The supported Velbus devices are:
+<!--
+The source of truth for this table is the
+`bundles/org.openhab.binding.velbus/src/main/resources/OH-INF/thing/thing-types.xml` file. The
+following xidel command can be used to regenerate the table from scratch:
 
-```text
-vmb1bl, vmb1bls, vmb1dm, vmb1led, vmb1ry, vmb1ryno, vmb1rynos, vmb1rys, vmb1ts, vmb2bl, vmb2ble, vmb2pbn, vmb4an, vmb4dc, vmb4ry, vmb4ryld, vmb4ryno, vmb6in, vmb6pbn, vmb7in, vmb8ir, vmb8pb, vmb8pbu, vmbdme, vmbdmi, vmbdmir, vmbel1, vmbel2, vmbel4, vmbelo, vmbelpir, vmbgp1, vmbgp2, vmbgp4, vmbgp4pir, vmbgpo, vmbgpod, vmbmeteo, vmbpirc, vmbpirm, vmbpiro, vmbvp1
 ```
-
-The type of a specific device can be found in the configuration section for things in the UI.
-It is part of the unique thing id which could look like:
-
-```text
-velbus:vmb4ryld:0424e5d2:01:CH1
+xidel -e \
+  'fn:for-each(//thing-type, function($thing) { "| " || $thing/@id || " | " || $thing/description || " |" })' \
+  src/main/resources/OH-INF/thing/thing-types.xml
 ```
-
-The thing type is the second string behind the first colon and in this example it is **vmb4ryld**.
+-->
+
+| Type     | Description |
```

Comment:
Can you format this table to look good in source mode too (for better readability)?